### ISSUE TITLE
BootUI - choose slot to boot

### ISSUE THREAD

**Issue by webmaster33** [chenxiaolong/DualBootPatcher#248]
On Samsung Galaxy S4 i9505, jfltexx:
The BootUI shows a counter, which counts down from 5 to 1.
I  can only Cancel booting, but can not choose, which slot to boot. 

Could be possible to choose slot, when I cancel booting?

**Comment by chenxiaolong** [comment]
Yep, once you cancel, go to "Switch ROM", which lets to boot into another ROM.

**Comment by Caio99BR** [comment]
`Could be possible to choose slot, when I cancel booting?`

You want much easy life, no?
